gradle/versions.gradle

```diff
@@ -31,6 +31,8 @@ dependencyManagement {
     dependency 'com.google.errorprone:error_prone_core:2.4.0'
     dependency 'com.google.errorprone:error_prone_test_helpers:2.4.0'
 
+    dependency 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Is this a requirement for Tuweni?  If not please remove.

Reply:
It is a requirement for Besu. It looks like those transitive dependencies are no longer there, and therefore it is required for Besu to explicitly state that dependency.

Reply:
It looks like it was added to the referencetests changes you are rolling back.  See if it works not adding this particular dependency once the reference tests are reset.

Reply:
It doesn't :(